unity-renderer/Assets/UIComponents/Scripts/Components/Carousel/CarouselComponentView.cs

```diff
@@ -247,6 +257,29 @@ public void SetManualControlsActive(bool isActived)
         dotsSelector.gameObject.SetActive(isActived && currentNumberOfItems > 1);
     }
 
+    public override void OnFocus()
+    {
+        base.OnFocus();
+
+        if (previousButton == null || nextButton == null || !showOnFocus)
+            return;
+
+        int currentNumberOfItems = itemsContainer.childCount;
+        previousButton.gameObject.SetActive(currentNumberOfItems > 1);
+        nextButton.gameObject.SetActive(currentNumberOfItems > 1);
+    }
+
+    public override void OnLoseFocus()
+    {
+        base.OnLoseFocus();
+
+        if (previousButton == null || nextButton == null || !showOnFocus)
+            return;
+
+        previousButton.gameObject.SetActive(false);
+        nextButton.gameObject.SetActive(false);
+    }
```

Comment:
**Question**: Are these new added behaviors approved by product? Because it is something that will also affect to the others carousels that we have in our app, like the one that we have in the explore menu.

Reply:
it's an approved behaviour, it doesn't affect other carousel because I added the boolean configuration `showOnFocus` defaulted to false so that previously existing carousels will work the same :)